packages/compass-connection-import-export/src/hooks/use-export.ts

```diff
@@ -59,6 +55,16 @@ export function useExportConnections(
   onChangeRemoveSecrets: (evt: React.ChangeEvent<HTMLInputElement>) => void;
   state: ExportConnectionsState;
 } {
+  const { favoriteConnections } = useConnectionRepository();
+  const connectionStorage = useConnectionStorageContext();
+  const exportConnectionsImpl =
+    connectionStorage.exportConnections?.bind(connectionStorage);
```

Comment:
do we need to bind the connectionStorage in this case?

Reply:
If not here I would've needed to do this while calling `onSubmit` so I decided to do it here.